rel-1.0.0/README.md

```diff
@@ -1,14 +1,88 @@
-.NET CLI Nightly Docker Image
+![](https://avatars0.githubusercontent.com/u/9141961?v=3&amp;s=100)
+
+.NET Core Nightly Docker Images
 ====================
 
-This repository contains the `Dockerfile` definition for Docker images that include the last-known-good (LKG) build of the next release of the [.NET Core command-line (CLI) tools](https://github.com/dotnet/cli).
+This repository contains `Dockerfile` definitions for Docker images that include last-known-good (LKG) builds of the next release of the [.NET Core command-line (CLI) tools](https://github.com/dotnet/cli).
 
 See [dotnet/dotnet-docker](https://github.com/dotnet/dotnet-docker) for images with official releases of [.NET Core](http://dotnet.github.io).
 
 This project is part of the .NET Core CLI tools. You can find samples, documentation, and getting started instructions at the [dotnet/cli] repo.
 
-# Supported tags and respective `Dockerfile` links
+[![Downloads from Docker Hub](https://img.shields.io/docker/pulls/microsoft/dotnet-nightly.svg)](https://registry.hub.docker.com/u/microsoft/dotnet-nightly)
+[![Stars on Docker Hub](https://img.shields.io/docker/stars/microsoft/dotnet-nightly.svg)](https://registry.hub.docker.com/u/microsoft/dotnet-nightly)
+
+
+## Supported tags
```

Comment:
>Supported tags [](start = 3, length = 14)

I think fully versioned tags need to exist for each of the images.  These are the tags that should be used in the onbuild, core images when referencing the base image.